scripts/docker_build.sh

```diff
@@ -21,4 +22,4 @@ GIT_ROOT=$(git rev-parse --show-toplevel)
 GIT_REVISION=$(git describe --abbrev=10 --always --tags --dirty)
 
 echo "Setting ORT_VERSION to $GIT_REVISION."
-DOCKER_BUILDKIT=1 docker build -f $GIT_ROOT/Dockerfile -t ort --build-arg ORT_VERSION=$GIT_REVISION $GIT_ROOT
+docker buildx build -f $GIT_ROOT/Dockerfile -t ort --build-arg ORT_VERSION=$GIT_REVISION --platform linux/amd64 $GIT_ROOT
```

Comment:
Please also update the README section about Docker if buildkit is not required anymore.

Reply:
Buildkit is still required, and `buildx` implies it.